### DISCUSSION TITLE
koreader opds menus don't appear on Android app

### DISCUSSION THREAD

**Discussion by abdelhalim** [koreader/koreader#12257] [Q&A] [answered]
same as title, koreader opds menus don't appear on Android app.
The plugin seems enabled.

Tested on Boox Page (Android 11) and Samsung S24 Ultra (Android 14)
Latest Apk from F-Droid store

Any hints will be appreciated.

**Comment by hius07** [comment]
It should appear when you are in the file browser, not in the reader.

**Reply by abdelhalim** [reply]
That was the issue. I found it at last.
Thanks a lot!!
I think this may deserve a mention in the docs.

Now, I was able to add my local OPDS, however connection fails silently from KOReader, however I can  connect from the same device through browser.

Is there any way to trouble shoot the issue?

**Reply by hius07** [reply]
Some response from the server can be found in koreader/crash.log.

**Reply by abdelhalim** [reply]
no crash logs. It just times out from KOReader, but works from browser

**Reply by hius07** [reply]
On android devices it is usually at storage/emulated/0/koreader/crash.log.

**Reply by abdelhalim** [reply]
this is where I looked, I don't see a crash.log

**Reply by hius07** [reply]
Cannot say anything without the log.
I remember some local servers (TinyOPDS) were not supported.

**Reply by hius07** [reply]
https://github.com/koreader/koreader/issues/6088

**Reply by abdelhalim** [reply]
I'm attempting to connect to Calibre

the KOReader app is installed on device Onyx Boox Page (Android 11)
Calibre is running on MacOS Air (latest software version).

I'll continue looking for the log.

**Reply by hius07** [reply, marked as answer]
Be sure your address looks like
http://192.168.1.1:8080/opds

**Reply by abdelhalim** [reply]
/opds was missing.

Calibre doesn't show this part when It provides the URL.

Great, that the issue.

Thanks a lot
